### PR TITLE
Added back absent mobile FAQ topics

### PR DESCRIPTION
**Pull request by cwarnermm**
Updated: 
- Install, Deploy, Upgrade, and Scale > Deploy Mattermost > Mobile Apps > Consider Mobile VPN Options
   - These links were attempting to point to content that was no longer in the codebase
   - Added this content back from another branch

Related to: https://github.com/mattermost/docs/pull/4828

### CODE REVIEW

File: source/deploy/mobile-faq.rst [hunk 1]
```diff
@@ -250,7 +250,60 @@ How do I connect users across internal and external networks?
...
+- Development server: ``api.development.push.apple.com:443``
+- Production server: ``api.push.apple.com:443``
+
+Run App Store versions of the Mattermost mobile apps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use the mobile applications hosted by Mattermost in the `Apple App Store <https://apps.apple.com/ca/app/mattermost/id1257222717>`_ or `Google Play Store <https://play.google.com/store/apps/details?id=com.mattermost.rn>`_ and connect with the `Mattermost Hosted Push Notification Service (HPNS) <https://docs.mattermost.com/deploy/mobile-hpns.html>` through your corporate proxy.
+
+.. note::
+  The use of hosted applications by Mattermost `can be deployed with Enterprise Mobility Management solutions via `AppConfig <https://docs.mattermost.com/deploy/mobile-appconfig.html>`__ but wrapping is not supported. See the `product documentation <https://docs.mattermost.com/deploy/deploy-mobile-apps-using-emm-provider.html#manage-app-configuration-using-appconfig>`__ for details.
```
Comment (cwarnermm) [inline source/deploy/mobile-faq.rst:309]:
```suggestion
  The use of hosted applications by Mattermost `can be deployed with Enterprise Mobility Management solutions via AppConfig <https://docs.mattermost.com/deploy/mobile-appconfig.html>`__ but wrapping is not supported. See the `product documentation <https://docs.mattermost.com/deploy/deploy-mobile-apps-using-emm-provider.html#manage-app-configuration-using-appconfig>`__ for details.
```

File: source/deploy/consider-mobile-vpn-options.rst [hunk 2]
```diff
@@ -45,7 +45,7 @@ Connect via Corporate Proxy Server
...
 
-- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
-- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://docs.mattermost.com/mobile/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment>`__
-- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://docs.mattermost.com/mobile/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server>`__
-- `Run App Store versions of the Mattermost Mobile apps <https://docs.mattermost.com/mobile/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`__
+- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/deploy/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
```
Comment (cwarnermm) [inline source/deploy/consider-mobile-vpn-options.rst:48]:
```suggestion
- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/deploy/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
```

File: source/deploy/consider-mobile-vpn-options.rst [hunk 3]
```diff
@@ -45,7 +45,7 @@ Connect via Corporate Proxy Server
...
+- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://docs.mattermost.com/deploy/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment>`__
```
Comment (cwarnermm) [inline source/deploy/consider-mobile-vpn-options.rst:49]:
```suggestion
- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://docs.mattermost.com/deploy/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment>`__
```

File: source/deploy/consider-mobile-vpn-options.rst [hunk 4]
```diff
@@ -45,7 +45,7 @@ Connect via Corporate Proxy Server
...
+- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://docs.mattermost.com/deploy/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server>`__
```
Comment (cwarnermm) [inline source/deploy/consider-mobile-vpn-options.rst:50]:
```suggestion
- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://docs.mattermost.com/deploy/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server>`__
```

File: source/deploy/consider-mobile-vpn-options.rst [hunk 5]
```diff
@@ -45,7 +45,7 @@ Connect via Corporate Proxy Server
...
+- `Run App Store versions of the Mattermost Mobile apps <https://docs.mattermost.com/deploy/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`__
```
Comment (cwarnermm) [inline source/deploy/consider-mobile-vpn-options.rst:51]:
```suggestion
- `Run App Store versions of the Mattermost Mobile apps <https://docs.mattermost.com/deploy/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`__
```

**Review by justinegeffen — APPROVED** [review]
These look good to me! :)

**Pull request merged**